doc-content/enterprise-only/employee-rostering/er-contracts-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='er-contracts-proc']
+= Entering the list of contracts
+
+You must enter the list of all types of contracts that the business uses for employees.
+
+A contract determines the maximum time that the employee can work in a day, calendar week, calendar month, or calendar year.
+
+You can set any of the limitations or none at all is a contract. For example, a part-time employee might not be allowed to work more than 20 hours per week, while a full-time employee might be limited to 10 hours in a day and 1800 hours during a year. Another contract might include no limitations on worked hours.
```

Comment:
Rethink. I think this is missing something or has something extra:
You can set any of the limitations or none at all is a contract.

For readability and esp for translations, consider using the same term or phrase here:
more than 20 hours in a week, while a full-time employee might be limited to 10 hours in a day and 1800 hours in a year.  Your way is more interesting to read, but...